waterdata/commands/lookup_generation/__init__.py

```diff
@@ -89,6 +88,66 @@
               'aw': {'name': 'USGS Active Groundwater Level Network Site'},
               'ad': {'name': 'USGS Annual Water Data Reports Site'}}
 
+US_STATES = [{'name': 'Alabama', 'abbreviation': 'AL'},
```

Comment:
Rather the generate a separate lookup file, should we just put this in constants.py?

Reply:
I just moved it there this morning. :stuck_out_tongue_closed_eyes: